test/libsolidity/semanticTests/operators/custom/redefine_operator_bind_in_derived_contract.sol

```diff
@@ -0,0 +1,29 @@
+type Int is int128;
+
+function add(Int, Int) pure returns (Int) {
+    return Int.wrap(3);
+}
+
+function another_add(Int, Int) pure returns (Int) {
+    return Int.wrap(7);
+}
+
+contract B {
+    using {add as +} for Int;
+
+    function f() pure public returns (Int) {
+        return Int.wrap(0) + Int.wrap(0);
+    }
+}
+
+contract C is B {
+    using {another_add as +} for Int;
```

Comment:
Maybe I missed that, but would it be possible to also create a test where the operator is not redefined here? Or will it just not work, because operators are not inherited?

Reply:
Ok, just found the test - so operators are not inherited :)